src/main/java/jenkins/metrics/api/MetricsRootAction.java

```diff
@@ -158,7 +158,7 @@ public HttpResponse doHealthcheck(StaplerRequest req, @QueryParameter("key") Str
             key = getKeyFromAuthorizationHeader(req);
         }
         Metrics.checkAccessKeyHealthCheck(key);
-        return Metrics.cors(key, new HealthCheckResponse(Metrics.healthCheckRegistry().runHealthChecks()));
+        return Metrics.cors(key, new HealthCheckResponse(Metrics.getHealthCheckResults()));
```

Comment:
was that really the intent? they are already being ran every minute
The problem I hit in #18 was very confusing because the periodic checks were not running but the API was returning correct results because they get executed in a completely different way.